source/apiVolontaria/volunteer/factories.py

```diff
@@ -0,0 +1,57 @@
+# coding: utf-8
+
+import factory
+import factory.fuzzy
+
+from .models import *
+from location.factories import AddressFactory
+from apiVolontaria.factories import UserFactory
+from faker import Faker
+
+fake = Faker()
+
+
+class CellFactory(factory.DjangoModelFactory):
+    class Meta:
+        model = Cell
+
+    name = fake.lexify(text="????????")
+    address = AddressFactory()
+
+
+class CycleFactory(factory.DjangoModelFactory):
+    class Meta:
+        model = Cycle
+
+    name = fake.lexify(text="????????")
+
+
+class TaskTypeFactory(factory.DjangoModelFactory):
+    class Meta:
+        model = TaskType
+
+    name = fake.lexify(text="????????")
+
+
+class EventFactory(factory.DjangoModelFactory):
+
+    class Meta:
+        model = Event
+
+    start_date = factory.LazyFunction(timezone.now)
+    end_date = factory.fuzzy.FuzzyDateTime(
+        timezone.now(), timezone.now() + timezone.timedelta(minutes=100))
```

Comment:
Please use a more readable form like :
```
end_date = factory.fuzzy.FuzzyDateTime(
    timezone.now(), 
    timezone.now() + timezone.timedelta(minutes=100),
)
```
**PS:** you will notice the comma on the last line

**PS2:** It's a better practice to make just one call to `timezone.now()` to be sure of the duration (so maybe create a new method in the class and call this method to populate the field :+1: